### DISCUSSION TITLE
[Projects Beta] Lack of Navigation UI

### DISCUSSION THREAD

**Discussion by FolkLoreee** [community/community#16826] [Projects and Issues]
Hi! The beta Projects table format looks excellent! 
However, unlike the Kanban format, the table display leaves out the navigation tabs (not sure if it's a bug), which makes it difficult to navigate back to the repository. Hope the GH team can work on this if other users echo similar feedback. Thanks!

**Comment by WisTex** [comment]
I agree. There are way too many clicks to get back to the repository. And too many clicks to get to the projects.

It would be nice for single repository projects, that you have direct links back and forth, without an intermediate screen.

For example:

- If the project is tied to one repository, have a direct link back to the repository from the project.
- If the repository is tied to one project, have a direct link from the repository to that project.
- Still provide a way to add more projects or repositories, as the case may be.

The UI for projects that have multiple repositories may be a bit more complicated, but ideally, it should be easy to get back to the repositories from the project's screens, and vice versa.

**Reply by baksale** [reply]
I could not find a way to navigate back from the new project view into the organization/repository other than through the browser url.

**Use Case:**
I'm in the project view and would like to create a new repository in the organization the project belongs.
Could not understand how this could be done easily.

**Comment by WisTex** [comment]
For projects that have multiple repositories, you could add a drop-down in the upper right-hand corner with all of the relevant repositories mentioned in the project, and clicking on the relevant repository sends you directly there.

For projects that have one repository, you could put a link or button that directly takes you back to the repository in the upper right-hand corner. No drop-down, just a direct link.

That way you can handle either use case.

Optional: Allow users to decide if they want the "back to repository" link to go to the main repository page or the issues page of that repository. I personally would want it to go to the issues page, but giving people a choice would be nice.

Note: the "back to repository" link should be the name of the repository, not the literal words "back to repository."

**Comment by ShadowCreator250** [comment]
Yes, I'm also missing this.

A Project (Beta) can either be owned by an organization or a profile. So I could imagine the respective name + tab-navigation (
for organization: Overview, Repositories, Projects, Packages, Teams, People, Settings | for profile: Overview, Repositories, Projects, Packages, Stars) or a shorter selection of that being shown above the project name. This would give a user a way back to the desired content.

This would match the current solution used in the Kanban boards and would maybe be easier to implement than the other suggestions that utilize the repository linking.

**Comment by omerbensaadon** [comment]
Hey @FolkLoreee! This was a bit different in legacy projects as they could belong to a single repository where-as projects beta projects belong to organizations. 

One potential solution would be to leverage the README on the projects (button, top right corner) and link to any repos you would like to access.

**Reply by omerbensaadon** [reply]
That being said, can understand the desire for better navigation within GitHub projects beta

**Reply by FolkLoreee** [reply]
Hi @omerbensaadon, thanks for the explanation! I think using the README as a means to navigate the Repositories within a project seems a little hacky, and members of this discussion have raised the same feedback about the lack of navigation. Hope the team can work on this if you think the issue is valid and is worth pursuing. Will leave the issue open for now until there's a clearer direction.
